test/snippet/io/alignment_file/emplace_back.cpp

```diff
@@ -1,12 +1,13 @@
-//! [all]
-#include <seqan3/io/alignment_file/all.hpp>
+#include <tuple>
 
-using namespace seqan3;
+#include <seqan3/io/alignment_file/output.hpp>
+#include <seqan3/std/filesystem>
 
 int main()
 {
     // I only want to print the mapping position (field::REF_OFFSET) and flag:
-    alignment_file_output fout{std::filesystem::temp_directory_path()/"my.sam", fields<field::REF_OFFSET, field::FLAG>{}};
+    seqan3::alignment_file_output fout{std::filesystem::temp_directory_path()/"my.sam",
```

Comment:
stream

Reply:
These snippets all fail automatic template type deduction when I switch from a file to a stream.

I can explicitly define the template in each of them, but I'm not sure if this changes the implication of the snippets too much.

Reply:
Nevermind, I just had to change the initialization a bit to make it work.